sql/mob_spawn_mods.sql

```diff
@@ -1561,6 +1561,12 @@ INSERT INTO `mob_spawn_mods` VALUES (17092904,1,1800,1); -- min gil 1800
 INSERT INTO `mob_spawn_mods` VALUES (17092904,2,5800,1); -- max gil 5800
 INSERT INTO `mob_spawn_mods` VALUES (17092904,15,2500,1); -- mug gil 2500
 
+-- Peg Powler
+-- Based on wiki (https://ffxiclopedia.fandom.com/wiki/Peg_Powler)
```

Comment:
These kinds of links typically go into the pull request comments.

Reply:
I put the link there to match the format with the rest of the file.